Main_Solution/bonus_task_test.rb

```diff
@@ -16,45 +19,40 @@ def setup
   end
 
   def test_hover
-
     link_to_profile = hover_on_avatar_of_user 1
-    assert(link_to_profile.displayed?)
+    expect(link_to_profile.displayed?).to be
```

Comment:
go through predicate matchers here http://www.relishapp.com/rspec/rspec-expectations/v/3-5/docs/built-in-matchers/predicate-matchers
this line can be changed to 

```expect(link_to_profile).to be_displayed```